carbonmark/components/pages/Project/Purchase/index.tsx

```diff
@@ -27,12 +27,12 @@ import { useState } from "react";
 import { FormValues, PurchaseForm } from "./PurchaseForm";
 import * as styles from "./styles";
 
-type Props = {
+export interface ProjectPurchasePageProps {
   project: Project;
-  listing: Listing;
-};
+  listing: Omit<Listing, "project">;
```

Comment:
I know you mentioned it, but still as you are using the `Omit` in multiple places, why not create a dedicated type after all?

Reply:
ok done